src/bootstrap/lib.rs

```diff
@@ -783,7 +783,7 @@ impl Build {
     fn cflags(&self, target: Interned<String>, which: GitRepo) -> Vec<String> {
         // Filter out -O and /O (the optimization flags) that we picked up from
         // cc-rs because the build scripts will determine that for themselves.
-        let mut base = self.cc[&target].args().iter()
+        let mut base: Vec<String> = self.cc[&target].args().iter()
```

Comment:
Can we have this in the collect annotation? It's also not clear to me exactly what's changing here, so maybe it's not worth it?

Reply:
Ah, this change was not really necessary but I wondered what the `.into()` was referring to (into what..) and that would have made it clear (into `String`) so I added it to make the code a bit clearer.

Since `base` is returned and the return type is `Vec<String>` the type is recognized from there perhaps.